modules/apps/static/portal-configuration/portal-configuration-persistence/src/main/java/com/liferay/portal/configuration/persistence/ConfigurationPersistenceManager.java

```diff
@@ -320,10 +322,19 @@ protected void doStore(
 
 		ConfigurationModelListener configurationModelListener = null;
 
-		if (hasPid(pid)) {
+		if (!pid.endsWith("factory") &&
+			(dictionary.get("_felix_.cm.newConfiguration") == null)) {
```

Comment:
@brianchandotcom @jorgeferrer I could not find a public constant for this String (it is a private field in `ConfigurationImpl`), nor any other way to test whether a configuration is new or not.  If you have any other suggestions I'm all ears :)